sql/src/test/java/io/crate/execution/engine/collect/files/FileReadingCollectorTest.java

```diff
@@ -196,14 +195,13 @@ public void testCollectWithEmptyLine() throws Throwable {
     public void testCollectWithOneSocketTimeout() throws Throwable {
         S3ObjectInputStream inputStream = mock(S3ObjectInputStream.class);
 
-        when(inputStream.read(new byte[anyInt()], anyInt(), anyByte()))
+        when(inputStream.read(any(), anyInt(), anyInt()))
```

Comment:
afaik you could also use `any(byte[].class)`

Reply:
it actually does.